DirectEmployers/MyJobs
# [PD-3448] cleanup sidebar component code

## What's this PR do?
PR cleans up the code inside of the sidebar component

## Screenshots (if appropriate)
N/A

## Where should the reviewer start?
Start inside of the analytics application

## How should this be manually tested?
Pull down repository. Go into any analytics application and make sure the sidebar is still functioning properly

## Any background context you want to provide?
